src/yunohost/domain.py

```diff
@@ -229,6 +237,63 @@ def domain_dns_conf(domain, ttl=None):
     return result
 
 
+@is_unit_operation()
+def domain_main_domain(operation_logger, new_main_domain=None):
```

Comment:
You might want to use this function now during the postinstall instead of `tools_maindomain` :wink: 

Also ideally we should rename the i18n key `log_tools_maindomain` to `log_domain_main_domain`

Reply:
> Also ideally we should rename the i18n key log_tools_maindomain to log_domain_main_domain

Isin't this one likely to "break" existing translations '-'?

Reply:
(done for the other one)

Reply:
>Isin't this one likely to "break" existing translations '-'?

Meh yes I suppose ... but is it really worth maintaining a legacy i18n string for an operation that happens basically only one or two times per installation é_è

Reply:
(done for the other one)

> > Isin't this one likely to "break" existing translations '-'?
> 
> Meh yes I suppose ... but is it really worth maintaining a legacy i18n string for an operation that happens basically only one or two times per installation é_è

Actually I think that should be ok if I change everything thinking about it, so let's try it :)

Reply:
Should be good :)

Reply:
Break existing translations how?

Reply:
By changing the keys but I don't think it will actually break it.